### ISSUE TITLE
Lag in scrolling through cards 

### ISSUE THREAD

**Issue by charlotteguard** [NWACus/avy#692]
If I'm swiping through cards quickly form the home screen the line around the polygon gets stuck so while the card is swiping the boarder remains the same.

**Comment by charlotteguard** [comment]
The video file is too large...looking for solutions

**Comment by charlotteguard** [comment]
I don't think it's that big of a deal - since it's probably pretty rare but worth addressing if we have extra time.

**Comment by stevekuznetsov** [comment]
Yep! For sure. Ideally we'd reconcile the current card state with the map eventually if we somehow miss the transition handler.

**Comment by stevekuznetsov** [comment]
This might be related to the throttle, xref #871
